### DISCUSSION TITLE
Allow custom local admin names

### DISCUSSION THREAD

**Discussion by Spencer021** [gameserverapp/Platform#549] [Suggestions]
Can you add the ability to use any local admin username? Having the local admin be default is asking for brute force attacks to happen, I know you have a bunch of other measures for security, so maybe even not allowing the local admin to be named "administrator" would enhance that as well. I have attached some references for guidance on this.

https://www.tenable.com/audits/items/CIS_Microsoft_Windows_Server_2016_STIG_v1.1.0_STIG_DC.audit:b0334b925aed090b58315101034f16f1

https://www.stigviewer.com/stig/microsoft_windows_server_2016/2022-03-01/finding/V-225026

https://www.stigviewer.com/stig/windows_server_2019/2019-12-12/finding/V-93281#:~:text=Configure%20the%20policy%20value%20for,name%20other%20than%20%22Administrator%22.

**Comment by mvaessen** [comment]
Hi, unfortunately are local admin accounts not suitable for installing everything that is required. With local admin accounts the install wont finish. Otherwise I would have gone for local admins as you suggest 👍
